testdata/packages/mssql-0.9.0/docs/README.md

```diff
@@ -0,0 +1,211 @@
+Note: This integration is in beta!
+
+The MSSQL integration allows to monitor the SQL server and fetch logs. 
+
+![metrics-dashboard](../img/metrics-dashboard.jpg "Metrics Dashboard")
+
+
+# Compatibility
+
+This version is tested with SQL Server 2017.
+
+# Configuration
+
+TODO: Add docs on how to configure SQL server if any special configs needed.
+
+If haven't setup the agent yet, checkout the docs here. To manually configure the agent, use the following configuration.
+```
+inputs:
+- type: log
+  paths: "/var/opt/mssql/log/error*"
+  pipeline: mssql-access-pipeline
+
+- type: metric/mssql
+  metricsets: ["transaction_log", "performance"]
+  period: 10s
+  enabled: true
+  hosts: ["sqlserver://sa@localhost"]
+```
+
+
+  
+# Logs
+
+| Field  Name               | Description                                             |
+|----------------------------|---------------------------------------------------------|
+| mssql.log.origin           | Origin of the log event                                 |
+| message                    | Message from the access log                             |
+
+An example event looks as following:
+
+<details>
```

Comment:
@dedemorton For showing example events, it would be nice to hide them by default but be able to expand it or similar.